<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1472</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="load_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Charger une image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download_button">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>670</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Télécharger</string>
    </property>
   </widget>
   <widget class="QComboBox" name="select_algo">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SLIC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SNIC</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="select_KX">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="select_KY">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QFrame" name="start_frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="start_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="fond_image_1" native="true">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>321</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <zorder>fond_image_1</zorder>
    <zorder>start_label</zorder>
   </widget>
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Segmenter</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="select_M">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QFrame" name="seg_frame">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="seg_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="fond_image_2" native="true">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>321</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <zorder>fond_image_2</zorder>
    <zorder>seg_label</zorder>
   </widget>
   <widget class="QSpinBox" name="select_SEUIL">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QComboBox" name="select_comp">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>450</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Palette couleur (256)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Palette couleur (65536)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fichier texte</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="compress_button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>500</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Compresser</string>
    </property>
   </widget>
   <widget class="QFrame" name="decomp_frame">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>290</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="decomp_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="fond_image_3" native="true">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>321</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <zorder>fond_image_3</zorder>
    <zorder>decomp_label</zorder>
   </widget>
   <widget class="QComboBox" name="select_comp_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>90</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Palette couleur (256)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Palette couleur (65536)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fichier texte</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="pal_path">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>130</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="index_path">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>160</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_path">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>220</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="indexbis_path">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>190</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="select_pal">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>120</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sélectionner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_index">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>150</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sélectionner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_indexbis">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sélectionner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_txt">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sélectionner</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>130</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Palette</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>160</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Index</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>190</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Index (bis)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>220</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fichier txt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decomp_button">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>250</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Décompresser</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>450</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithme de compression :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithme de segmentation :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Superpixels</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre à l'horizontal :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre à la verticale :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>471</x>
      <y>310</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>M :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seuil :</string>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>831</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PSNR_label">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>600</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="taux_label">
    <property name="geometry">
     <rect>
      <x>638</x>
      <y>530</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="select_format">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>670</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PPM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PNG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JPG</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="download_comp_button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>560</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Télécharger</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1472</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
